app/views/users/index.html.erb

```diff
@@ -1,3 +1,21 @@
-<h1>Users</h1>
-
-<p>List of users</p>
\ No newline at end of file
+<div class="main-container">
+	<div class="users-container">
+		<% if @users.blank? %>
+		<p>There are no users</p>
+		<% else %>
+		<% @users.each do |user| %>
+			<a href="/users/<%= user.id%>" class="user-container">
```

Comment:
- [ ] It would be better to use rails routing and navigations tag to change views. Let's fix that